src/lib.rs

```diff
@@ -459,6 +460,19 @@ impl Log for Logger {
     fn flush(&self) {}
 }
 
+impl fmt::Debug for Logger{
```

Comment:
There are some nice [helper methods on `Formatter`](https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.debug_struct) that can make this easier. So you could have something like:

```rust
f.debug_struct("Logger")
 .field("directives", &self.directives)
 .field("filter", &self.filter)
 .field("target", &self.target)
 .finish()
```

Reply:
Done :-)